hack/install-tools.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 
-set -e
+set -o errexit
+set -o nounset
+set -o pipefail
 
 STARTTIME=$(date +%s)
```

Comment:
It's used to print how long the script took, see the last line in the file. The time is not printed when the script fails though, but that has nothing to do with my PR ;)